api-driver/ApiDriver.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
```

Comment:
Don't open the PR into `master`, I cannot merge that. If I did, it could break customers' functionality. This repo is public and is used in modules.
We can merge it to `dev` and then create a new tag, once tested.